data/tests/tests_common.txt

```diff
@@ -583,6 +583,9 @@ test "Depart"
 		input
 		branch zoom_restore_loop
 			"test: Depart: zoom_restore_counter" > 0
+		# Clean-up temporary variables
+		apply
+			clear "test: Depart: zoom_restore_counter"
```

Comment:
`"clear <condition-name>"` sets the value of the named condition to 0, rather than eagerly it from the collection.

https://github.com/endless-sky/endless-sky/blob/58190d7be67e4261d3338f887bd4bd6f693a82e6/source/ConditionSet.cpp#L363-L364

This particular condition is already 0 here (or else we would still be looping), so what's the point?

Reply:
Sorry, I thought that `clear` mapped to `"unset"`, not to `= 0`. I should have looked a bit better.